packages/graphic-walker/src/fields/filterField/slider.tsx

```diff
@@ -222,12 +213,56 @@ const Slider: React.FC<SliderProps> = React.memo(function Slider ({
                             }
                         }}
                         style={{
-                            left: `${range[1] * 100}%`,
+                            left: `calc(${range[1] * 100}% - 1em)`,
                             transform: 'translate(0, -50%)',
                         }}
                     />
                 </SliderTrack>
             </SliderElement>
+            <div className="output">
+                <output htmlFor="slider:min">
+                    <div className="my-1">{t('filters.range.start_value')}</div>
+                    {
+                        <input
+                            type="text"
+                            className="block w-full rounded-md border-0 py-1 px-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-1 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
```

Comment:
mark

Reply:
i've encapsulated input to a separate component